app/scripts/modules/kubernetes/src/serverGroup/configure/configuration.service.js

```diff
@@ -172,7 +174,11 @@ module.exports = angular
       const validContainers = [];
       const invalidContainers = [];
       containers.forEach(function(container) {
-        if (container.imageDescription.fromContext || container.imageDescription.fromTrigger) {
+        if (
```

Comment:
Small style nit, this can be reduced to:

```javascript
const { fromContext, fromTrigger, fromArtifact } = container.imageDescription;
if (fromContext || fromTrigger || fromArtifact) {
```

Reply:
Good idea; made the change